CONTRIBUTING.md

```diff
@@ -2,6 +2,18 @@
 
 Contributions are always welcome, both reporting issues and submitting pull requests!
 
+### Setup
+
+Follow these steps to run the shopify_python tests:
+
+```bash
+pyenv local 3.6.1 # or whatever version you want to work with
+virtualenv ENV
```

Comment:
on second thought, this isn't how I usually create virtual environments anymore. Does this require anything installed locally first?

Reply:
I usually do `python -m venv /path/to/my/venvs/shopify_python` but open to alternatives.

Reply:
I've started using `pipenv` myself

Reply:
What do other open source repos suggest for getting started?